setup.py

```diff
@@ -0,0 +1,32 @@
+from setuptools import setup
+
+import os
+
+package_files = []
+for dir, subdirs, files in os.walk("schema"):
+    for d in subdirs:
+        package_files.append(dir + "/" + d)
+    for f in files:
+        package_files.append(dir + "/" + f)
+
+setup(name='lsst-alert-packet',
```

Comment:
Please move most of this to setup.cfg so that other tooling can find it.

Reply:
I'm curious: why setup.cfg instead of pyproject.toml, https://www.python.org/dev/peps/pep-0518/?

Reply:
Probably because I'm an old fuddy duddy who doesn't keep up with all the hip kids. My main point is to move as much of the static configuration code out of python and into configuration files.